importScripts/sql/testImports/locationHistory.sql

```diff
@@ -7,12 +7,15 @@ UNION(SELECT LocationHistoryID FROM LocationHistoryExport WHERE LocationStatus I
 UNION(SELECT LocationHistoryID FROM LocationHistoryExport WHERE PrimaryKey_Object_Table IS NOT NULL LIMIT 5)
 UNION(SELECT LocationHistoryID FROM LocationHistoryExport WHERE o_Accession_Full_ID IS NOT NULL LIMIT 5)
 UNION(SELECT LocationHistoryID FROM LocationHistoryExport WHERE o_ItemType IS NOT NULL LIMIT 5)
-UNION(SELECT LocationHistoryID FROM LocationHistoryExport WHERE topLevelLocation IS NOT NULL LIMIT 5)
```

Comment:
These fields no longer exist in the export view, as we're using the data that the client has cleaned instead.